epochStart/metachain/economics.go

```diff
@@ -222,19 +236,19 @@ func (e *economics) VerifyRewardsPerBlock(
 	receivedEconomics := metaBlock.EpochStart.Economics
 	if computedEconomics.TotalToDistribute.Cmp(receivedEconomics.TotalToDistribute) != 0 {
 		return fmt.Errorf("%w total to distribute computed %d received %d",
-			process.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalToDistribute, receivedEconomics.TotalToDistribute)
+			epochStart.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalToDistribute, receivedEconomics.TotalToDistribute)
 	}
 	if computedEconomics.TotalNewlyMinted.Cmp(receivedEconomics.TotalNewlyMinted) != 0 {
 		return fmt.Errorf("%w total newly minted computed %d received %d",
-			process.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalNewlyMinted, receivedEconomics.TotalNewlyMinted)
+			epochStart.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalNewlyMinted, receivedEconomics.TotalNewlyMinted)
 	}
 	if computedEconomics.TotalSupply.Cmp(receivedEconomics.TotalSupply) != 0 {
 		return fmt.Errorf("%w total supply computed %d received %d",
-			process.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalSupply, receivedEconomics.TotalSupply)
+			epochStart.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.TotalSupply, receivedEconomics.TotalSupply)
 	}
 	if computedEconomics.RewardsPerBlockPerNode.Cmp(receivedEconomics.RewardsPerBlockPerNode) != 0 {
 		return fmt.Errorf("%wrewards per block per node computed %d received %d",
-			process.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.RewardsPerBlockPerNode, receivedEconomics.RewardsPerBlockPerNode)
+			epochStart.ErrEndOfEpochEconomicsDataDoesNotMatch, computedEconomics.RewardsPerBlockPerNode, receivedEconomics.RewardsPerBlockPerNode)
 	}
 
```

Comment:
add check that previous epoch start metablock hash is the same.
Also add display economics.

Reply:
done